<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reader</class>
 <widget class="QDialog" name="reader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Imię</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="nameEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="surnameLabel">
     <property name="text">
      <string>Nazwisko</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="surnameEdit"/>
   </item>
   <item>
    <widget class="QRadioButton" name="maleRadio">
     <property name="text">
      <string>Mężczyzna</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="femaleRadio">
     <property name="text">
      <string>Kobieta</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="readerIdLabel">
     <property name="text">
      <string>Numer czytelnika: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="confirmButton">
     <property name="text">
      <string>Zatwierdź</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Anuluj</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
